Sources/Tuvi.Core.DataStorage.Impl/DataStorage.cs

```diff
@@ -1549,8 +1511,8 @@ private void BuildContact(SQLiteConnection connection, Contact contact, Cancella
             if (contact.LastMessageData != null)
             {
                 cancellationToken.ThrowIfCancellationRequested();
-                var emailData = GetEmailAddressData(connection, contact.LastMessageData.AccountEmailId);
-                contact.LastMessageData.AccountEmail = emailData.ToEmailAddress();
+                var account = connection.Find<Account>(contact.LastMessageData.AccountId);
+                contact.LastMessageData.AccountEmail = account?.Email;
```

Comment:
Why didn't you check if the account is not null in other places?

Reply:
Fixed.